remake.yml

```diff
@@ -31,11 +38,24 @@ targets:
     command: find_oldest_sites(states, parameter)
 
   # TODO: PULL SITE DATA HERE
-  state_tasks:
+  state_combiners:
     command: do_state_tasks(oldest_active_sites, '1_fetch/src/get_site_data.R', '3_visualize/src/plot_site_data.R', '2_process/src/tally_site_obs.R')
```

Comment:
I was expecting to see `123_state_tasks.R` added here because 123_state_tasks.yml will now need this file to be able to run `combine_obs_tallies` and `summarize_timeseries_plots`. Maybe the build worked for you because you had defined/sourced those functions locally in your R session, but I'm guessing (and feel free to check and correct me if I'm wrong!) that if you started a new R session and tried to force rebuild those combiner targets, you'd get an error if you didn't add the dependency here.

Reply:
Hmmm....it still seems to work after clearing the local environment 🤷‍♀️

Reply:
And you used `force=TRUE`?

Reply:
yes

Reply:
I think the build will work in either case, because `123_state_tasks.R` is still in `sources` of the parent `remake.yml` file. So it gets sourced and the functions within it are available. 

I don't think you need to include it here to get the target to build, but you do need to include it here in order for the `do_state_tasks` to recognize changes to the combiner function _within_ that file. Otherwise you could change the way things are combined, rebuild this target, and remake/scipiper would falsely think it was up to date. 

But since `do_state_tasks` spins off its own separate remakefile (123_state_tasks.yml) Alison and I were both likely expecting that functions sourced from the remake.yml wouldn't be visible to targets within the new yml, since they aren't explicitly connected...but seems that isn't the case. 

I'd think that clearing your env and running `scmake(remake_file = '123_state_tasks.yml')` _will_ trigger the error Alison mentioned above - but that would be because you are running that `scmake` totally outside of the parent `remake.yml`.

Reply:
Ahh, that makes sense, @jread-usgs...and it might even continue to work if calling `scmake(remake_file = '123_state_tasks.yml')` in a clean session because that file `include`s *remake.yml*....BUT I still think you should have it both places, @collnell, as a matter of principle: a YAML should have as `sources` all those code files that are used within it, both because that's usually what's needed for **scipiper** to work and also because it makes the pipeline easier to read (because others know where to look to find out which code files matter).